README.md

```diff
@@ -37,10 +37,12 @@ Our general DEV workflow should be the next:
 2 - Make sure all the info you need to develop it is available to you, if not ask the team.<br />
 3 - assign the task to yourself and set it on the "To do" or "Doing" columns depending on when you will actually start.<br />
 4 - create a new branch from the 'develop' branch named this way:<br />
-&nbsp;  'feature/description' for additions <br />
-&nbsp;  'bugfix/description' for bugfixes <br />
-&nbsp;  'enhancement/description' for improvements <br />
-&nbsp;  'issue/description' <br />
+```
+'feature/description' for additions
+'bugfix/description' for bugfixes
+'enhancement/description' for improvements 
+'issue/description' for issues
+```
```

Comment:
Please change this section to follow format:

`feature/<description>`:  Add new features.

Reply:
Done!